src/libs/ReportUtils.ts

```diff
@@ -7677,26 +7677,37 @@ function hasUpdatedTotal(report: OnyxInputOrEntry<Report>, policy: OnyxInputOrEn
 /**
  * Return held and full amount formatted with used currency
  */
-function getNonHeldAndFullAmount(iouReport: OnyxEntry<Report>, policy: OnyxEntry<Policy>): string[] {
+function getNonHeldAndFullAmount(iouReport: OnyxEntry<Report>, policy: OnyxEntry<Policy>, isReimbursableOnly: boolean): string[] {
```

Comment:
Rename prop to `shouldExcludeNonReimbursables`

Reply:
Done